### PR TITLE
Binarized neural network blog from Kunal

### PR DESCRIPTION
**Pull request by kunalgoyal9**

### CODE REVIEW

File: source/_posts/binarized-neural-network.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Binary Neural Network “A New Revolution in AI”?
```
Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:2]: use 3-4 some keyword from this list 

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

File: source/_posts/binarized-neural-network.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+date: 2020-09-31 08:00:00
+author: Kunal Goyal
+category: AI
+tags:
```
Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:6]: Use 1-2 long tail keyword from the list

File: source/_posts/binarized-neural-network.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:1]: Link to previous content - in content there should be link with previous content

Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:1]: Keyword should be linked with website - Means it redirected to main website

Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:1]: Content should have link to - Redirect on previous blog link also

Comment (Rajeev111974) [inline source/_posts/binarized-neural-network.md:1]: Image should be optimized with Alt text

Comment (kunalgoyal9) [reply source/_posts/binarized-neural-network.md:1]: No previous blog can be linked to

File: source/_posts/binarized-neural-network.md (hunk 1 above)
Comment (kunalgoyal9) [reply source/_posts/binarized-neural-network.md:2]: This is the research blog... I have added Nayan and AI keyword in it

**Review by Rajeev111974 — APPROVED** [review]

**Pull request merged**